taxonomies/categories.txt

```diff
@@ -32805,7 +32802,7 @@ ciqual_food_name:fr:Yaourt, lait fermenté ou spécialité laitière, aromatisé
 
 <en:Yogurts
 <en:Fermented milk drinks
-en:Drinkable yogurts
+en:Drinkable yogurts, yogurt-drink
```

Comment:
we canonicalize "yogurt drink" to yogurt-drink, but in the synonyms, we should list the correct name : " yogurt drink" (with accents, without the dashes)

Reply:
OK, although I'm not sure I understand what you mean by "canonicalize" in this context :)